### ISSUE TITLE
Deprecation Warning during testing tfjs-core

### ISSUE THREAD

**Issue by usmanmuhd** [tensorflow/tfjs#1312]
#### TensorFlow.js version
Master branch of tfjs-core

#### Describe the problem or feature request
When running `yarn test` it gives a warning
```
WARN: 'tf.batchNormalization() is going away. Use tf.batchNorm() instead, and note the positional argument change of scale, offset, and varianceEpsilon You can disable deprecation warnings with tf.disableDeprecationWarnings().'
```
I would work on updating the tests if deemed appropriate.

**Comment by dsmilkov** [comment]
This is intentional since we decided to keep the old batchNormalization in 1.0 and we will have to unit test it.
